voltixApp/VoltixApp/Model/Chain.swift

```diff
@@ -70,6 +70,31 @@ enum Chain: String, Codable, Hashable, CaseIterable {
         }
     }
 
+    var asset: String {
```

Comment:
why `.solana` cause a fatal error?  is it specific for swap? maybe we should name it like `swapAsset` ?

Reply:
you could raise follow up PR to address it